### PR TITLE
AP_AHRS: correct infinite recursion in AP_AHRS_View

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by magicrub** [comment]
Looks good to me

File: libraries/AP_AHRS/AP_AHRS_View.h [hunk 1]
```diff
@@ -118,12 +118,12 @@ class AP_AHRS_View
         return ahrs.get_relative_position_NE_origin(vecNE);
     }
     
-    bool get_relative_position_D_home(float &posD) const {
-        return get_relative_position_D_home(posD);
+    void get_relative_position_D_home(float &posD) const {
+        return ahrs.get_relative_position_D_home(posD);
```
Comment (OXINARF) [inline libraries/AP_AHRS/AP_AHRS_View.h]: I don't really like to do a return from a void method, but this is a bit nitpicking. Other than this looks good.

**Comment by tridge** [comment]
nice catch! How did you find this?

**Comment by tridge** [comment]
pushed, with return void fixed

**Comment by peterbarker** [comment]
@tridge clang++ picked it up, and I thought, "that's probably a bad thing in an autopilot" as it scrolled past :-)

**Pull request closed without merging**